src/Model/MyParcelCustomsItem.php

```diff
@@ -49,7 +52,11 @@ public function getDescription()
      */
     public function setDescription($description)
     {
-        $this->description = $description;
+        /**
+         * Description cut after 47 chars
+         */
+
+        $this->description = Str::limit($description, self::DESCRIPTION_MAX_LENGTH);
```

Comment:
In WooCommerce hadden we nog `...` op het einde toegevoegd moeten we dat hier ook doen ?

Reply:
deleted

Reply:
deze ... wordt door ::limit vanzelf daar gezet.

Reply:
Goed gebruik gemaakt van de Laravel helper 👍